lib/ui/views/grade_details_view.dart

```diff
@@ -131,7 +131,7 @@ class _GradesDetailsViewState extends State<GradesDetailsView>
     if (model.isBusy) {
       return const Center(child: CircularProgressIndicator());
     } else if (model.course.inReviewPeriod &&
-        !(model.course.reviewCompleted ?? true)) {
+        !(model.course.allReviewsCompleted ?? true)) {
```

Comment:
```suggestion
        !model.course.allReviewsCompleted ?? false) {
```
It seems more readable

Reply:
Hmm, I don't think that the following expressions are the equivalent:
```
// Initial
!(model.course.allReviewsCompleted ?? true)

// Suggested
!model.course.allReviewsCompleted ?? false
``` 

In the suggested change, we can't negate `model.course.allReviewsCompleted` with `!` since it could be `null`.

In the first expression we're negating `(model.course.allReviewsCompleted ?? true)`. So in the event of a `null` value we're assigning to `true` and then negating it. `!true == false`.


*Same thing for the following suggested changes*